hikari/guilds.py

```diff
@@ -299,6 +299,14 @@ class Member(users.User):
     user: users.User = attr.ib(repr=True)
     """This member's corresponding user object."""
 
+    @property
+    def _dm_channel(self) -> typing.Optional[channels_.DMChannel]:
```

Comment:
Should we be accessing private attributes like this? Would it not make more sense for member’s important of send and fetch DM channel To just call self.user.send and self.user.fetch_dm_channel